app/views/rooms/calendar.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%=  @room.name %></h1>
+
+<h2><%=  @month %> <%=  @year %></h2>
+<td class="td-btn"><%= link_to 'Create new Event', new_event_path(id: @room.id), class: "btn btn-secondary link-btn" %></td>
+<%= week_calendar(number_of_weeks: 1, events: @events) do |date, events| %>
```

Comment:
This currently **crashes** when, logging in, adding an event for today with a room set and opening the calendar for that room!